frontend/src/scenes/session-recordings/player/PlayerSidebar.tsx

```diff
@@ -43,6 +43,11 @@ export function PlayerSidebar(): JSX.Element {
         SessionRecordingSidebarTab.NETWORK_WATERFALL,
     ]
 
+    // Show AI summary tab in the second position if the flag is enabled
+    if (featureFlags[FEATURE_FLAGS.AI_SESSION_SUMMARY]) {
+        sidebarTabs.splice(1, 0, SessionRecordingSidebarTab.SESSION_SUMMARY)
```

Comment:
amusingly https://github.com/PostHog/posthog/blob/improve-quality-of-single-replay-summary/frontend/src/lib/utils.tsx#L178 is making the title read `Ai session summary` when really it should be `AI session summary`

not urgent to fix but definitely need to sort at some point

Reply:
Yeah, I started reworking it and then stopped myself with "I get too deep, it's good enough, iterate later" :)